src/layout.js

```diff
@@ -375,15 +375,23 @@ export function applyStaticLayout(element) {
 
   // Input layout attributes.
   const inputLayout = layoutAttr ? parseLayout(layoutAttr) : null;
-  userAssert(inputLayout !== undefined, 'Unknown layout: %s', layoutAttr);
+  userAssert(
+    inputLayout !== undefined,
+    'Invalid "layout" value: %s',
```

Comment:
Is it relatively trivial to include in this error message the name of the offending component?

Reply:
Added the element itself. 👍